boards/iot-lab_M3/auto_init_ng_netif/netif_board.c

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_iot-lab_M3
+ * @{
+ *
+ * @file
+ * @brief       Network device initialization code
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "board.h"
+#include "auto_init.h"
+#include "ng_at86rf2xx.h"
+#include "net/ng_nomac.h"
+#include "net/ng_netbase.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+/**
+ * @brief   Define stack parameters for the MAC layer thread
+ * @{
+ */
+#define MAC_STACKSIZE           (KERNEL_CONF_STACKSIZE_DEFAULT)
+#define MAC_PRIO                (0U)
+/** @} */
+
+/**
+ * @brief   Device descriptor for the Atmel radio
+ */
+static ng_at86rf2xx_t radio;
+
+/**
+ * @brief   Stack for the MAC layer thread
+ */
+static char nomac_stack[MAC_STACKSIZE];
+
+
+void auto_init_ng_netif(void)
```

Comment:
but this does syntactical not make sense: the notion is: `auto_init_MODULE_NAME`, and the module is called `ng_netif`...